index.js

```diff
@@ -18,26 +18,41 @@ module.exports = function (opts) {
 		args.push('/s', opts.system, '/u', opts.username, '/p', opts.password);
 	}
 
-	if (Array.isArray(opts.filter) && opts.filter.length) {
+	if (opts.apps) {
+		args.push('/apps');
+	}
+
+	if (Array.isArray(opts.filter)) {
```

Comment:
I prefer to keep it `Array` only. I don't like mixing types with little benefit.